src/Memoization.cpp

```diff
@@ -74,8 +74,14 @@ class FindParameterDependencies : public IRGraphVisitor {
 
     void visit(const Variable *var) override {
         if (var->param.defined()) {
-            record(var->param);
+            if (var->param.is_buffer() &&
+                var->type.is_int_or_uint()) {
```

Comment:
Would it be a bit more explicit to exclude handles (is that what the type of the pointer is?) rather than include int/uint?

Reply:
Done.